docs/static/monitoring/monitoring-apis.asciidoc

```diff
@@ -58,7 +58,7 @@ You can also require HTTP Basic authentication by setting `api.auth.type: basic`
 --------------------------------------------------
 api.auth.type: basic
 api.auth.basic.username: "logstash"
-api.auth.basic.password: "stashy"
+api.auth.basic.password: "s3cUreP4$$w0rD"
```

Comment:
@roaksoax we with @karenzone didn't make a decision to include a note/statement for password policy (or improving LS security) today but I will think about it. Hope current change unblocks the FF.